### PR TITLE
Abyss.AbyssOverlay version 1.2.2

### PR DESCRIPTION
**Pull request by TakenMC**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/18016)

### CODE REVIEW

File: manifests/a/Abyss/AbyssOverlay/1.2.2/Abyss.AbyssOverlay.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Abyss.AbyssOverlay
+PackageVersion: 1.2.2
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/Chit132/abyss-overlay/releases/download/v1.2.2/Abyss.Overlay.Setup.v1.2.2.exe
+  InstallerSha256: 471D8FDCF6903A72C5FDFB76F49923BD2DEA3166D4CD462F76560B4D65D227C5
```
Comment (OfficialEsco) [inline manifests/a/Abyss/AbyssOverlay/1.2.2/Abyss.AbyssOverlay.installer.yaml:9]:
```suggestion
- Architecture: neutral
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/Chit132/abyss-overlay/releases/download/v1.2.2/Abyss.Overlay.Setup.v1.2.2.exe
  InstallerSha256: 471D8FDCF6903A72C5FDFB76F49923BD2DEA3166D4CD462F76560B4D65D227C5
  InstallerSwitches:
    Custom: /NORESTART /CURRENTUSER
- Architecture: neutral
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/Chit132/abyss-overlay/releases/download/v1.2.2/Abyss.Overlay.Setup.v1.2.2.exe
  InstallerSha256: 471D8FDCF6903A72C5FDFB76F49923BD2DEA3166D4CD462F76560B4D65D227C5
  InstallerSwitches:
    Custom: /NORESTART /ALLUSERS
```

For compatability.

File: manifests/a/Abyss/AbyssOverlay/1.2.2/Abyss.AbyssOverlay.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Abyss.AbyssOverlay
+PackageVersion: 1.2.2
+PackageLocale: en-US
+Publisher: Abyss
```
Comment (OfficialEsco) [inline manifests/a/Abyss/AbyssOverlay/1.2.2/Abyss.AbyssOverlay.locale.en-US.yaml:6]:
```suggestion
Publisher: Chit
Author: Abyss
```
Needs to match what control panel sees

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by TakenMC** [comment]
Sorry I forgot
I approve the changes and you can merge it

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You need to commit the changes.

**Comment by TakenMC** [comment]
There updated

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**